modules/serverless-rn-1-35-0.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies
+//
+// * about/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-35-0_{context}"]
+= Red Hat {ServerlessProductName} 1.35
+
+{ServerlessProductName} 1.35 is now available. New features, updates, and known issues that pertain to {ServerlessProductName} on {ocp-product-title} are included in the following notes:
+
+[id="new-features-1-35-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.15.
+* {ServerlessProductName} now uses Knative Eventing 1.15.
+* {ServerlessProductName} now uses Kourier 1.15.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.15.
+* {ServerlessProductName} now uses Knative for Apache Kafka 1.15.
+* The `kn func` CLI plugin now uses `func` 1.16.
+
+* Go functions using S2I builder are now available as a Generally Available (GA) feature for Linux and Mac developers, allowing them to implement and build Go functions on these platforms.
+
+* It is now possible to automatically discover and register `EventTypes` based on the structure of incoming events, simplifying the overall configuration and management of `EventTypes`.
+
+* Knative Eventing catalog is now available in OpenShift Developer Console (ODC). You can explore the catalog to discover different event types, along with their descriptions and associated metadata, making it easier to understand the system capabilities and functionalities.
+
+* Knative Eventing now supports long-running background jobs. This feature separates resource-intensive or time-consuming tasks from the primary event processing flow, boosting application responsiveness and scalability.
+
+* Knative Eventing catalog is now available in Backstage. Backstage is as an internal developer portal, centralizing tools and resources.
```

Comment:
```suggestion
* Knative Eventing catalog is now available in OpenShift Developer Console (ODC). You can explore the catalog to discover different event types, along with their descriptions and associated metadata, making it easier to understand the system capabilities and functionalities.

* Knative Eventing catalog is now available in Backstage. Backstage is as an internal developer portal, centralizing tools and resources.

* Knative Eventing now supports long-running background jobs. This feature separates resource-intensive or time-consuming tasks from the primary event processing flow, boosting application responsiveness and scalability.
```

Just a re-order of the items to "group" related ones

Reply:
Done!